client/src/components/Preview/Preview.js

```diff
@@ -123,6 +125,7 @@ class Preview extends Component {
           <div className="btn-toolbar">
             {this.renderBackButton()}
             {this.buildToolbarButtons()}
+            <ViewModeToggle id="view-mode-toggle-in-preview-nb" area={'preview'} />
```

Comment:
Ideally we'd use the injector to inject a ViewModeToggle prop into Preview which would allow devs to customise this component in the Preview implementation if they wanted to. For example:

```js
import { inject } from 'lib/Injector';

// ...

render() {
  const { ViewModeComponent } = this.props;
  return (
    // ..
    <ViewModeComponent { ...props } />

// ...

Preview.propTypes = {
  ViewModeComponent: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
};

// ...

export default inject(
  ['ViewModeToggle'],
  (ViewModeToggle) => ({
    ViewModeComponent: ViewModeToggle
  }),
  'Admin.Preview'
);
```

Reply:
You'll also need to register the component with Injector in `src/boot/registerComponents.js`